app/notify/notify_email/candidate_booking_confirmation.482c3495-8e99-46f0-93d4-b88e31879565.md

```diff
@@ -0,0 +1,32 @@
+Dear ((candidate_name)),
+
+Here are some details about your school experience placement at ((school_name)).
+
+* School or college: ((school_name))
+* Address: ((school_address))
+* Placement dates: ((placement_start_date)) to ((placement_finish_date))
+* Start and finish times: ((school_start_time)) to ((school_finish_time))
+* Dress code: ((school_dress_code))
+* Parking: ((school_parking))
+
+If you have any questions about your bookings:
+
+Name: ((school_admin_name))
+Email address: ((school_admin_email))
+UK telephone number: ((school_admin_telephone))
+
+On the day of your school experience placement you'll need to report to:
+
+Name: ((school_teacher_name))
+Email address: ((school_teacher_email))
+UK telephone number: ((school_teacher_telephone))
+
+You can also contact them if you have any questions about your actual placement.
+
+# Placement details
+
+((placement_details))
+
+# Remember to bring
+* your DBS certificate
+* your £((placement_fee)) placement fee
```

Comment:
Thoughts on moving this whole sentence into a personalisation so we can leave it out if the school doesn't charge a placement fee

Reply:
Yep, I wondered this too. Opted for directly copying @fredbrenton's templates for the time being.